superset/translations/fr/LC_MESSAGES/messages.json

```diff
@@ -3877,7 +3877,27 @@
       ],
       "Scope": ["Périmètre"],
       "Dependent on": ["Dépend de"],
-      "No matching records found": ["Aucun résultat trouvé"]
+      "No matching records found": ["Aucun résultat trouvé"],
```

Comment:
isn't the `.json` file generated here?  shouldn't this be in the `.po` file?

Reply:
Yes indeed! I thought that was covered in this PR, but that doesn't seem to be the case.